examples/python/sample-job.py

```diff
@@ -0,0 +1,91 @@
+#!/usr/bin/env python3
+
+import argparse
+from kubernetes import config, client
+
+# create_job.py
+# This example will demonstrate full steps to submit a Job.
+
+# Make sure your cluster is running!
+config.load_kube_config()
+crd_api = client.CustomObjectsApi()
+api_client = crd_api.api_client
+
+
+def get_parser():
+    parser = argparse.ArgumentParser(
+        description="Submit Kueue Job Example",
+        formatter_class=argparse.RawTextHelpFormatter,
+    )
+    parser.add_argument(
+        "--job-name",
+        help="generateName field to set for job",
+        default="sample-job-",
+    )
+    parser.add_argument(
+        "--image",
+        help="container image to use",
+        default="gcr.io/k8s-staging-perf-tests/sleep:v0.0.3",
+    )
+    parser.add_argument(
+        "--args",
+        nargs="+",
+        help="args for container",
+        default=["30s"],
+    )
+    return parser
+
+
+def generate_job_crd(job_name, image, args):
+    """
+    Generate an equivalent job CRD to sample-job.yaml
+    """
+    metadata = client.V1ObjectMeta(
+        generate_name=job_name, labels={"kueue.x-k8s.io/queue-name": "user-queue"}
+    )
+
+    # Job container
+    container = client.V1Container(
+        image=image,
+        name="dummy-job",
+        args=args,
+        resources={
+            "requests": {
```

Comment:
@vsoch I think Aldo meant this.

Reply:
This is what I was thinking - https://github.com/kubernetes-sigs/kueue/pull/950#discussion_r1254927598 - do you think that’s ok?